docs/production-deployment/cloud/worker-health.mdx

```diff
@@ -129,19 +128,20 @@ sum(increase(temporal_activity_schedule_to_start_latency_seconds_count[5m])) by
 
 **Target**
 
-This latency should obviously be a very low value - close to zero. Anything else, indicates bottlenecking.
+This latency should be a very low value -- close to zero.
+Anything else indicates bottlenecking.
 
 ### Sync Match Rate {#sync-match-rate}
 
 The Sync Match Rate measures the rate of Tasks that can be delivered to Workers without having to be persisted (Workers are up and available to pick them up) to the rate of all delivered Tasks.
 
 **Calculate Sync Match Rate**
 
-$$
-temporal\_cloud\_v0\_poll\_success\_sync\_count / temporal\_cloud\_v0\_poll\_success\_count = N%
-$$
+```
+temporal_cloud_v0_poll_success_sync_count ÷ temporal_cloud_v0_poll_success_count = N
+```
```

Comment:
Isn't $$ the indicator for in-line equations? This may be intentional math.

Reply:
Yes, but it broke the page width and wasn't really rendering properly, nor was it used elsewhere on this same page, so I think it was a non-working convention.